pkg/server/problem_ranges.go

```diff
@@ -289,7 +289,7 @@ const debugProblemRangesTemplate = `
         <DIV CLASS="row">
           {{- if $.LeaderNotLeaseholder}}
             {{- range $_, $r := $.LeaderNotLeaseholder}}
-              <a href="/debug/range?id={{$r}}">{{$r}}</a>
+              <a href="/debug/range?id={{$r}}">{{$r}}</a> &nbsp;
```

Comment:
Sadly, going with just `{{" "}}` doesn't produce any spaces either.  I think the `{{- ...` is eating it as well.

So with `<a href="/debug/range?id={{$r}}">{{$r}}</a>{{" "}}`
We get:
```
              <a href="/debug/range?id=0">0</a> 
              <a href="/debug/range?id=1">1</a> 
              <a href="/debug/range?id=2">2</a> 
   ```
But no spaces between ids.

![screen shot 2017-05-01 at 11 30 00](https://cloud.githubusercontent.com/assets/1614265/25583837/8f6c3682-2e61-11e7-958e-17c3c1f35fee.png)


I think the nbsp is correct because there's only one anchor per line it is allowed to break on at the newline instead of on the nbsp itself.